providers/nexmo/README.md

```diff
@@ -0,0 +1,15 @@
+# Notifire Nexmo Provider
+
+A Nexmo sns provider library for [@notifire/core](https://github.com/notifirehq/notifire)
```

Comment:
Actually `sns` seems like the boilerplate bug @scopsy

Reply:
@ranrib on it, thanks!